docs/user-guide/release-notes/2.7.x.rst

```diff
@@ -46,6 +69,13 @@ Known Issues
 Client Changes
 --------------
 
+* Admin and consumer Pulp clients now support -v and -vv flags to get additional
```

Comment:
I am not sure if this is the place, but we could add info about the possibility of retrieving info about specific repo, since lately I've seen a lot of users asking about this feature.
https://pulp.plan.io/issues/225